### PR TITLE
Cleanup for data input.

### PR DESCRIPTION
**Pull request by nc-maen**
Long description and default path in default_paths.py.
The input data format
---------------------
'path' is the path where all file data is assumed to be in
'input' is a dictionary with information of where to find the tables:
'PSRM Afregning', 'Afregning_Underretning', 'Udligning', 'Udtræk'.

Each value in the dictornary is a list of length 2.
First value is a file name or a list of file names.
Second value is a sheet name.

If multiple file names are given as a value then the program will concatinate
the content to one table.

Multiple Excel file or sheets are currently not implemented.

The particular data locations
---------------------------

We have so far recieved updated data multiple times which we organized i
version folders. Running this file as a script should test all the locations.
Typical directory structure looks like this:

Data/
├── v1
│   ├── PSRM_CI_FT_BASE.xlsx
├── v2
│   ├── Afregninger.xlsx
│   ├── CI_FT_BASE.xlsx
│   ├── Udligninger.xlsx
├── v3
│   ├── Afregninger.xlsx
│   ├── CI_FT_BASE.xlsx
│   ├── Udligninger.xlsx
│   ├── Udtræk.xlsx
├── v4
│      ├── 3977.txt
│      ├── 3982.txt
│    ├── Afregninger.xlsx
│    ├── CI_FT_BASE.xlsx
│    ├── Test.txt
│    ├── Udligninger.xlsx

Notes
------

'EXTERNAL_OBLIGATION_ID' is not found in v1 of udtraeksdata

### CODE REVIEW

**Review by nc-ejp — APPROVED** [review]
Godt arbejde

File: psrm_ci_ft_base.py [hunk 1]
```diff
@@ -9,58 +9,33 @@
 
 class PSRM_CI_FT_BASE:
 
-    def __init__(self, path, fname='PSRM_CI_FT_BASE.xlsx', multi_sheets=None):
+    def __init__(self, path=None, input=None):
```
Comment (nc-dtan) [inline psrm_ci_ft_base.py:12]: Har du rettet eksemplet nede i bunden af denne fil?

**Review by nc-dtan — CHANGES_REQUESTED** [review]

File: psrm_ci_ft_base.py (hunk 1 above)
Comment (nc-maen) [reply psrm_ci_ft_base.py:12]: aah, nej det gør jeg lige.

Comment (nc-maen) [reply psrm_ci_ft_base.py:12]: Det fikset nu! Kan jeg merge nu?

**Review by nc-dtan — APPROVED** [review]

**Pull request merged**